src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -168,6 +191,9 @@ namespace AppInstaller::CLI::Execution
         // Set the context to the terminated state.
         void Terminate(HRESULT hr, std::string_view file = {}, size_t line = {});
 
+        // Resumes the context
+        void Resume() { m_terminationHR = S_OK; m_isTerminated = false; }

```

Comment:
>Resume [](start = 13, length = 6)

I think there is a better way to handle this with the existing tools.  For instance, completion creates a `Clone()` of the `Context` and disables telemetry while it is running.  Disabling telemetry might be too extreme, but we do want a way to indicate that a single update has failed, not the entire run.

So I would suggest cloning the `Context`, adding a new member that indicates this is a subset of the operation, and proceeding with each update as a singular operation on the clone.  Then we can pass the "subset" flag to the telemetry call on failure and have an easy way to separate them in the stream.